lib/Runtime/Language/DynamicProfileInfo.h

```diff
@@ -500,6 +500,7 @@ namespace Js
             bool disableObjTypeSpec_jitLoopBody : 1;
             bool disablePowIntIntTypeSpec : 1;
             bool disableLoopImplicitCallInfo : 1;
+			bool disableStackArgOpt : 1;
```

Comment:
Done.

---
In reply to: [67793943](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67793943) [](ancestors = 67793943)